.circleci/config.yml

```diff
@@ -137,8 +137,8 @@ jobs:
     steps:
       - setup_vm
       - run:
-          name: 'Install Protobuf'
-          command: pip3 install --user protobuf
+          name: 'Install Validator Dependencies'
+          command: ./.circleci/install_dependencies.sh
```

Comment:
Very nice! This should be `install_validator_dependencies.sh`. (Eagerly waiting to see if this works 😃 )

Reply:
Done